### ISSUE TITLE
Matching edge indexes in viewport

### ISSUE THREAD

**Issue by Neacroon** [addam/Export-Paper-Model-from-Blender#66]
In dealing with extremly complicated models, thousands of faces, it would be very helpful to look up exact position on the screen or printed views, to match all the angles, not just right edges. Is it possible to implement a script that will re-index edge indices, to match those in viewport? 

And after that, would it be possible to export ortho views with those numers attached?
End result I look for is like this pdf:
[RZUTY.pdf](https://github.com/addam/Export-Paper-Model-from-Blender/files/3024163/RZUTY.pdf)

**Comment by Treehee** [comment]
Wouldn't it then be more useful to straight up print the angles on the edges? Like negative degrees for mountain folds and positive for valley or smth.

**Comment by Neacroon** [comment]
Indeed, that would have been most useful too. Nevertheless, plans and views would greatly simplify working with them - welding it all together later (laser cut metal plates, complex).

**Comment by addam** [comment]
A little thing you can do is to save the UV layout during export and use "Sync UV Selection" to find the edges in 3d. It's far from perfect but it has helped me many times -- I have made some complex models myself.

If the edges were numbered as you suggest, how would you find an edge in the ortho views? I'm afraid you'd have to search through the whole image... that seems quite too slow.
